individual/workflows/example_import_individual_workflow.py

```diff
@@ -14,7 +14,7 @@
 _dob_field = 'data_de_nascimento'
 
 
-def import_individual_workflow(*args, user_uuid=None, upload_uuid=None, **kwargs):
+def example_import_individual_workflow(*args, user_uuid=None, upload_uuid=None, **kwargs):
```

Comment:
Since whole function relies on transaction.atomic a wrapper could be used. However it is not an issue, just a matter of more readable code.